vmdb/app/controllers/provider_foreman_controller.rb

```diff
@@ -71,14 +71,23 @@ def refresh
   def provision
     assert_privileges("provider_foreman_provision")
     @prov_id = find_checked_items
-    @prov_id = params[:id] if @prov_id.empty?
-
-    render :update do |page|
-      page.redirect_to :controller     => "miq_request",
-                       :action         => "prov_edit",
-                       :prov_id        => @prov_id,
-                       :org_controller => "configured_system",
-                       :escape         => false
+    @prov_id.push(params[:id]) if @prov_id.empty?
```

Comment:
@brandondunne Yes, needs to be a plural. I can make that change and push an update once you comment on my question below.
```find_checked_items``` returns an array.